test/execute-functional-tests-cloud.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------
+# Copyright 2023 The Radius Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+# Make sure that you have default environment in the default group.
+# The way to check that is to run the following commands:
+# `rad group switch default`
+# `rad env list` and check if there is an environment named `default`.
+# If not, create one using `rad env create default`.
+
+# To run functional tests, you have to be in `kind-radius` group and
+# `kind-radius` environment within the `default` group.
+# You can switch to the group using `rad group switch kind-radius`
+# and then switch to the environment using `rad env switch kind-radius`.
+
+rad group create default
+rad env create kind-radius
+rad group create kind-radius
```

Comment:
Why does it need to create two groups?

Reply:
Please see: https://github.com/search?q=repo%3Aradius-project%2Fradius+%22kind-radius%22&type=code.

Reply:
The link you referenced points to all occurrences of `kind-radius` - can you help me understand how does that result into requiring two groups to be created?

Reply:
For example, in this [test](https://github.com/radius-project/radius/blob/main/test/functional-portable/corerp/noncloud/resources/recipe_terraform_test.go#L64) we have hard-coded the group called kind-radius. And if you run the functional tests in your local without creating kind-radius group, you will see that some tests will fail.

Yes, the link I referenced points to all occurrences of `kind-radius`. If you go over the list, you will see that most of them are in the functional tests' territory.